vulture_os/toolkit/api_parser/crowdstrike/crowdstrike.py

```diff
@@ -24,13 +24,11 @@
 __doc__ = 'CrowdStrike API Parser toolkit'
 __parser__ = 'CROWDSTRIKE'
 
-import datetime
-import json
 import logging
-import time
+from json import dumps
 from datetime import timedelta
+from requests import session, exceptions
 
```

Comment:
Prefer importing the exceptions directly to avoid reader confusion with `exceptions` objects that could come from anywhere

```suggestion
from requests import session
from requests.exceptions import ConnectionError, ReadTimeout

```